test/byContract/ENSIntegration/ENSIntegration.test.ts

```diff
@@ -0,0 +1,126 @@
+// SPDX-FileCopyrightText: 2020-2024 IEXEC BLOCKCHAIN TECH <contact@iex.ec>
```

Comment:
why did you change the name of the file with `test.` ?

Reply:
I thought doing the same on a previous PR but abandoned it, since we don't do it for each test file => or we should set a naming file convention

Reply:
Using `.test` prefix seems to be the convention.
I'm preparing a PR to rename other files https://github.com/iExecBlockchainComputing/PoCo/pull/106

Reply:
https://jestjs.io/docs/configuration#testregex-string--arraystring